api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -207,24 +203,56 @@ private Workspace createWorkspaceImpl(Workspace workspace) {
     validateWorkspaceApiModel(workspace);
 
     DbCdrVersion cdrVersion = getLiveCdrVersionId(workspace.getCdrVersionId());
-
+    DbAccessTier accessTier = cdrVersion.getAccessTier();
     DbUser user = userProvider.get();
-    final DbBillingProjectBufferEntry bufferedBillingProject;
```

Comment:
this short block is now `claimBillingProject()` - part of an effort to refactor common code between `createWorkspace()` and `cloneWorkspace()`